src/test/java/co/com/api/test/ApiHttpBinTest.java

```diff
@@ -9,162 +13,158 @@
 
-    }
+  }
 
-    @Test
-    @DisplayName("Check get response and validate property")
-    public void apiGetWithQueryParamsTest() {
-
-        given()
-
-                .queryParam("name", "John")
-                .queryParam("age", "31")
-                .queryParam("city", "New York")
-                .when()
-                .contentType(ContentType.JSON)
-                .get("/get")
-                .then()
-                .statusCode(HttpStatus.SC_OK)
-                .body("args.name", equalTo("John"))
-                .body("args.age", equalTo("31"))
-                .body("args.city", equalTo("New York"));
-
-
-    }
-
-    @Test
-    @DisplayName("Consume services with POST")
-
-    public void apiPostTest() {
-
-        PersonDto payload = PersonDto.builder()
-                .name("John")
-                .age("31")
-                .city("New York")
-                .build();
-
-
-        given()
-                .when()
-                .contentType(ContentType.JSON)
-                .when()
-                .body(payload)
-                .post("/post")
-                .then()
-                .statusCode(HttpStatus.SC_OK);
-
-    }
-
-    @Test
-    @DisplayName("Consume services with Delete")
-
-    public void apiDeleteTest() {
-
-        PersonDto payload = PersonDto.builder()
-                .name("John")
-                .age("31")
-                .city("New York")
-                .build();
-
-        given()
-                .when()
-                .contentType(ContentType.JSON)
-                .when()
-                .body(payload)
-                .delete("/delete")
-                .then()
-                .statusCode(HttpStatus.SC_OK);
-    }
-
-    @Test
-    @DisplayName("Consume services with Patch")
-
-    public void apiPatchTest() {
-
-        PersonDto payload = PersonDto.builder()
-                .name("John")
-                .age("31")
-                .city("New York")
-                .build();
-
-        given()
-                .when()
-                .contentType(ContentType.JSON)
-                .when()
-                .body(payload)
-                .patch("/patch")
-                .then()
-                .statusCode(HttpStatus.SC_OK);
-
-    }
-
-    @Test
-    @DisplayName("Consume services with PUT")
-
-    public void apiPutTest() {
-
-        PersonDto payload = PersonDto.builder()
-                .name("John")
-                .age("31")
-                .city("New York")
-                .build();
-
-        given()
-                .when()
-                .contentType(ContentType.JSON)
-                .when()
-                .body(payload)
-                .when()
-                .put("/put")
-                .then()
-                .statusCode(HttpStatus.SC_OK);
-    }
+  @Test
+  @DisplayName("Check get response and validate property")
+  public void apiGetWithQueryParamsTest() {
+
+    given()
+
+          .queryParam("name", "John")
```

Comment:
this indentation has 6 spaces from the base, can you configure being 2 spaces?

Reply:
I reduce de spaces for indent the code, in the last commit